cmp/tab/TabContainerModel.js

```diff
@@ -83,7 +83,7 @@ export class TabContainerModel {
         if (isEmpty(tabs)) {
             tabs.push({
                 content: () => div({
-                    className: 'xh-text-color-accent xh-pad',
+                    className: 'xh-text-color-accent xh-pad xh-empty-text-tab',
```

Comment:
Obv could target this container by adding a class to the tabcontainer and targeting:

`.my-tab-container {
.xh-text-color-accent.xh-pad {
     display: flex;
     justify-content: center;
}
}`

but this is too general, and would match anything we might want to use these xh classes on that isnt this div